### PR TITLE
[ODS-4872] Investigate and Fix Issues with SDK

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs [hunk 1]
```diff
@@ -145,7 +145,7 @@ private Operation CreateGetOperation(OpenApiMetadataPathsResource openApiMetadat
                     _contentTypeStrategy.GetOperationContentType(openApiMetadataResource, ContentTypeUsage.Readable)
                 },
                 parameters = CreateGetByExampleParameters(openApiMetadataResource, isCompositeContext),
-                responses = CreateReadResponses(openApiMetadataResource)
+                responses = CreateReadResponses(openApiMetadataResource,true)
```
Comment (tbmeador-dlp) [inline Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs:148]:
```suggestion
                responses = CreateReadResponses(openApiMetadataResource, true)
```

Comment (tbmeador-dlp) [inline Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs:148]:
```suggestion
                responses = CreateReadResponses(openApiMetadataResource, true)
```

File: Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs [hunk 2]
```diff
@@ -191,15 +191,15 @@ private Operation CreateGetByIdOperation(OpenApiMetadataPathsResource openApiMet
                     _contentTypeStrategy.GetOperationContentType(openApiMetadataResource, ContentTypeUsage.Readable)
                 },
                 parameters = parameters,
-                responses = CreateReadResponses(openApiMetadataResource)
+                responses = CreateReadResponses(openApiMetadataResource ,false)
```
Comment (tbmeador-dlp) [inline Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs:194]:
```suggestion
                responses = CreateReadResponses(openApiMetadataResource, false)
```

File: Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs [hunk 3]
```diff
@@ -191,15 +191,15 @@ private Operation CreateGetByIdOperation(OpenApiMetadataPathsResource openApiMet
...
             };
         }
 
-        private Dictionary<string, Response> CreateReadResponses(OpenApiMetadataPathsResource openApiMetadataResource)
+        private Dictionary<string, Response> CreateReadResponses(OpenApiMetadataPathsResource openApiMetadataResource ,bool isArray = false)
```
Comment (tbmeador-dlp) [inline Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs:198]: I don't see any reason to have a default value here.
```suggestion
        private Dictionary<string, Response> CreateReadResponses(OpenApiMetadataPathsResource openApiMetadataResource, bool isArray)
```

**Review by tbmeador-dlp — CHANGES_REQUESTED** [review]

**Comment by vimayya** [comment]
@semalaiappan can you update the SDK reference in EdFi.LoadTools.Test project?

**Review by tbmeador-dlp — APPROVED** [review]

**Pull request merged**